public-api/src/main/java/org/pmiops/workbench/cdr/model/AchillesResult.java

```diff
@@ -0,0 +1,192 @@
+        this.analysis = analysis;
+    }
+    public AchillesResult analysis(AchillesAnalysis analysis) {
+        this.analysis = analysis;
+        return this;
+    }
+
+    @Column(name="stratum_1")
+    public String getStratum1() {
+        return stratum1;
+    }
+    public void setStratum1(String stratum1) {
+        this.stratum1 = stratum1;
+    }
+    public AchillesResult stratum1(String val) {
+        this.stratum1 = val;
+        return this;
+    }
+
+    @Column(name="stratum_2")
+    public String getStratum2() {
+        return stratum2;
+    }
+    public void setStratum2(String stratum2) {
+        this.stratum2 = stratum2;
+    }
+    public AchillesResult stratum2(String val) {
+        this.stratum2 = val;
+        return this;
+    }
+
+    @Column(name="stratum_3")
+    public String getStratum3() {
+        return stratum3;
+    }
+    public void setStratum3(String stratum3) {
+        this.stratum3 = stratum3;
+    }
+    public AchillesResult stratum3(String val) {
+        this.stratum3 = val;
+        return this;
+    }
+
+    @Column(name="stratum_4")
+    public String getStratum4() {
+        return stratum4;
+    }
+    public void setStratum4(String stratum4) {
+        this.stratum4 = stratum4;
+    }
+    public AchillesResult stratum4(String val) {
+        this.stratum4 = val;
+        return this;
+    }
+
+    @Column(name="stratum_5")
+    public String getStratum5() {
+        return stratum5;
+    }
+    public void setStratum5(String stratum5) {
+        this.stratum5 = stratum5;
+    }
+    public AchillesResult stratum5(String val) {
+        this.stratum5 = val;
+        return this;
+    }
+
+    @Transient
+    public String getStratum5Name() {
+        return stratum5Name;
+    }
+    public void setStratum5Name(String stratum5Name) {
+        this.stratum5Name = stratum5Name;
+    }
+    public AchillesResult stratum5Name(String val) {
+        this.stratum5Name = val;
+        return this;
+    }
+
+    @Column(name="count_value")
+    public Long getCountValue() {
+        return countValue;
+    }
+    public void setCountValue(Long countValue) {
+        this.countValue = countValue;
+    }
+    public AchillesResult countValue(Long val) {
+        this.countValue = val;
+        return this;
+    }
+
+    @Override
+    public String toString() {
+        return "AchillesResult{" +
+                "id=" + id +
+                ", analysisId=" + analysisId +
+                ", analysis=" + analysis +
+                ", stratum1='" + stratum1 + '\'' +
+                ", stratum2='" + stratum2 + '\'' +
+                ", stratum3='" + stratum3 + '\'' +
+                ", stratum4='" + stratum4 + '\'' +
+                ", stratum5='" + stratum5 + '\'' +
+                ", countValue=" + countValue +
+                ", stratum5Name='" + stratum5Name + '\'' +
+                '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
nit: This would be cleaner using java.util.Objects like so:
if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        AchillesResult that = (AchillesResult) o;
        return id == that.id &&
                analysisId == that.analysisId &&
                Objects.equals(analysis, that. analysis) &&
                Objects.equals(stratum1, that. stratum1) &&
                Objects.equals(stratum2, that.stratum2) &&
                Objects.equals(stratum3, that. stratum3) &&
                Objects.equals(stratum4, that. stratum4) &&
                Objects.equals(stratum5, that. stratum5) &&
                Objects.equals(countValue, that. countValue);

Reply:
Thanks . I'll fix it.

Reply:
Ok. just pushed that. I'll check codacy.